bucket/firefox-developer.json

```diff
@@ -30,6 +30,7 @@
             "Firefox Developer Edition"
         ]
     ],
+    "persist": "distribution",
```

Comment:
Should we persist profile here as well and in the others?

I noticed firefox.json persists profile

Reply:
this PR for persist 'distribution' only.

Reply:
`profile` folder only exists in portable version. It is designated in shortcut argument in the manifest.

https://github.com/ScoopInstaller/Extras/blob/a9301ccd584b3f5b20d2fe03fcdf9c4f2cfab0a9/bucket/firefox-portable.json#L21